tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "outDir": "lib",
     "sourceMap": true,
     "strict": true,
-    "strictNullChecks": true,
+    "strictNullChecks": false,
```

Comment:
I know you mentioned you had to turn `strictNullChecks` off but I'd like to pair to see if we can keep it on